### DISCUSSION TITLE
How to stop automatic update on Win 7 ?

### DISCUSSION THREAD

**Discussion by dewcansam** [VSCodium/vscodium#1836] [General]
So I install the 1.81 version for win 7 and have no problem there. I can get to settings and turn off automatic update. But it has already downloaded it and it just keeps pestering me to apply the update. Is there a way to remove the downloaded update ? Or stop / block the notification ? Or is there a switch to use on install to prevent automatic download ?

**Comment by dewcansam** [comment]
NM I think I just got it.

**Comment by daiyam** [comment]
`"update.mode": "none",` in your settings
